src/main/java/org/spongepowered/common/mixin/api/mcp/entity/monster/GuardianEntityMixin_API.java

```diff
@@ -27,19 +27,23 @@
 import net.minecraft.entity.LivingEntity;
 import net.minecraft.entity.monster.GuardianEntity;
 import net.minecraft.network.datasync.DataParameter;
+import org.spongepowered.api.data.value.Value;
 import org.spongepowered.api.entity.living.Living;
 import org.spongepowered.api.entity.living.monster.guardian.Guardian;
 import org.spongepowered.asm.mixin.Final;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.Shadow;
 
+import javax.annotation.Nullable;
 import java.util.Optional;
+import java.util.Set;
 
 @Mixin(GuardianEntity.class)
 public abstract class GuardianEntityMixin_API extends MonsterEntityMixin_API implements Guardian {
 
     @Shadow @Final private static DataParameter<Integer> TARGET_ENTITY;
     @Shadow private void setTargetedEntity(int entityId) { } // setTargetedEntity
+    @Shadow @Nullable public abstract LivingEntity shadow$getTargetedEntity();
```

Comment:
Remove any unused shadows you find. Also remove this one as you don't use it anymore.